packaging_assets/package.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+# Copyright 2022 Contributors to the Parsec project.
+# SPDX-License-Identifier: Apache-2.0
+
+# Create a quickstart package
+
+ARCH="x86"
+OS="linux"
+PACKAGE_PATH=$(pwd)
+WORK_DIR="/tmp/workdir-parsec/"
+JOBS_NUMBER=1
+SCRIPT_PATH=$(realpath $0)
+ASSETS_DIR=$(dirname $SCRIPT_PATH)
+PARSEC_DIR=$(dirname $ASSETS_DIR)
+VERSION=$(cargo metadata --format-version=1 --no-deps --offline | python3 -c "import sys, json; data=json.load(sys.stdin); print([pkg['version'] for pkg in data['packages'] if pkg['name'] == 'parsec-service'][0])")
+
+# Usage
+USAGE_STR=\
+"Usage:\n"\
+"package.sh [Options]\n"\
+"Options:\n"\
+" -o {path}: Output absolute path, the default path is the current directory i.e. $(pwd)\n"\
+" -j {jobs}: Number of parallel jobs, Default is $JOBS_NUMBER"\
+" -h : Display this help menu\n"
+
+# Flags
+while getopts v:o:j:h flag
+do
+    case "${flag}" in
+        o) PACKAGE_PATH=${OPTARG};;
+        j) JOBS_NUMBER=${OPTARG};;
+        h) echo -e $USAGE_STR; exit 0;;
+    esac
+done
+
+check_mandatory_flags() {
+    echo "Checking mandatoy flags..."
+    if [ -z "$VERSION" ];then
+            echo 'Missing the version please run with -h for help' >&2
+            exit 1
+    fi
+}
+
+check_release_tag() {
```

Comment:
I don't quite understand we try to archive with this function.
For the package name we use the version number we get from metadata anyway.
So, instead of checking for latest tags I think we need to either:
1. Check that the current tag is equal to the version from metadata or
2. ignore tags and include the current commit hash into the package name in addition to the version.

Reply:
Since it's a manual step, then by mistake a package might upload which isn't from the latest release.